### PR TITLE
fix: [CDS-38231]: added releaserepolistview

### PR DESCRIPTION
**Pull request by sahithikolichala**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger jest

**Review by tan-nhu — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger typecheck

File: src/modules/70-pipeline/components/ManifestSelection/ManifestInterface.ts [hunk 1]
```diff
@@ -63,6 +63,19 @@ export interface ManifestListViewProps {
...
+  updateStage: (stage: StageElementConfig) => Promise<void>
+  stage: StageElementWrapper | undefined
+  isPropagating?: boolean
+  connectors: PageConnectorResponse | undefined
+  refetchConnectors: () => void
+  listOfManifests: Array<any>
+  isReadonly: boolean
+  deploymentType: ServiceDefinition['type']
+  allowableTypes: MultiTypeInputType[]
+  allowOnlyOne?: boolean
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestInterface.ts:76]: Please move this out to separate interface

File: src/modules/70-pipeline/components/ManifestSelection/ManifestListView/ManifestListView.tsx [hunk 2]
```diff
@@ -92,7 +92,11 @@ import HelmWithOCI from '../ManifestWizardSteps/HelmWithOCI/HelmWithOCI'
 import { getConnectorPath } from '../ManifestWizardSteps/ManifestUtils'
 import css from '../ManifestSelection.module.scss'
 
-const showAddManifestBtn = (isReadonly: boolean, allowOnlyOne: boolean, listOfManifests: Array<any>): boolean => {
+export const showAddManifestBtn = (
+  isReadonly: boolean,
+  allowOnlyOne: boolean,
+  listOfManifests: Array<any>
+): boolean => {
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestListView/ManifestListView.tsx:99]: Why is this exported for use in gitops manifest..

**Review by bhavya-sinha — CHANGES_REQUESTED** [review]

**Pull request merged**